src/reduced_basis/rb_scm_evaluation.C

```diff
@@ -534,17 +530,13 @@ void RBSCMEvaluation::legacy_read_offline_data_from_files(const std::string & di
   // Resize C_J based on C_J_stability_vector and Q_a
   C_J.resize( C_J_length );
   for (std::size_t i=0; i<C_J.size(); i++)
-    {
-      RBParameters::const_iterator it     = get_parameters().begin();
-      RBParameters::const_iterator it_end = get_parameters().end();
-      for ( ; it != it_end; ++it)
-        {
-          std::string param_name = it->first;
-          Real param_value;
-          C_J_in >> param_value;
-          C_J[i].set_value(param_name, param_value);
-        }
-    }
+    for (const auto & pr : get_parameters())
+      {
+        std::string param_name = pr.first;
```

Comment:
Should this be another one we change to const reference while we're at it?

Reply:
Yep, good catch.

Reply:
Oh no, wait a minute. The problem here was that `C_J_in` in this code chunk does not have an `>>` overload for const references if I recall correctly?

Reply:
Ah, misread similarly named parameters. I will change that.